sections/Chapter 5 Sprint 3/planning.tex

```diff
@@ -0,0 +1,48 @@
+\section{Sprint Planning}
+\subsection{Knox Goal}
+The overall goal for the Knox project during this sprint was to get the search engine to work. To accomplish this goal the product owner specified a list of expectation that when completed would fufill a minimal viable product. The specified list is outlined in figure X.
+\begin{itemize}
+	\item It should be possible to make searches in all the available databases.
+	\item It should be possible to make searches using keywords in all the available data.
+	\item It should be possible to make searches in all Nordjyske articles from 2017-2021
+	\item It should be possible to make searches in all the Grundfos manuals, barring those that have shown to be problematic during processing.
+	\item The UI should minimaly be able to display a link to the given article but preferably dispaly the title and text.
+\end{itemize}
+
+With the overall goal for the sprint contained in figure x, a granulated sprint goal could be made, that takes into account the focus of our scrum group.
+
+
+\textbf{Sprint goal}
+As noted in the preceding sprint, the code base that was inherited from the previous iteration was deemed problematic and costly to maintain. Therefore our sprint goal for this iteration was based on updating the code base to better comply with the requirements for the Knox goal.
```

Comment:
```suggestion
As noted in the preceding sprint, the codebase that was inherited from the previous iteration was deemed problematic and costly to maintain. Therefore our sprint goal for this iteration was based on updating the codebase to better comply with the requirements for the Knox goal.
```

Reply:
Generally, I would like to know how all this ties together with the search engine